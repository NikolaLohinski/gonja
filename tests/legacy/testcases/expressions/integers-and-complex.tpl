{{ 10-100 }}
{{ -(10-100) }}
{{ -(-(10-100)) }}
{{ -(-(10-100)) ** 2 }}
{{ -1 * (-(-(10-100))) }}
{{ -1 * (-(-(10-100)) ** 2) ** 3 + 3 * (5 - 17) + 1 + 2 }}
