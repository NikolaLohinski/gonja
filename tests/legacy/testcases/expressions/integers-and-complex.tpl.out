-90
90
-90
-8100.0
90
531440999967.0
